core/services/keystore/aptos.go

```diff
@@ -0,0 +1,182 @@
+package keystore
```

Comment:
@george-dorin what is the latest re: keystore as a loopp?

is the pattern here still the correct thing to do?

Reply:
This pattern is also used for starknet which is our latest BIX integration going to mainnet soon 2 months ago

https://github.com/smartcontractkit/chainlink/pull/12575

@jmank88 mentioned something about how keystore will be extracted as a loopp in the future, maybe he can chime in here as well :)

Reply:
AFAIK following local style is still the correct thing to do.